test/e2etest/test-config.tfvars.tmpl

```diff
@@ -86,12 +87,14 @@ jira_reserved_code_cache = "512m"
 jira_db_instance_class       = "db.t3.micro"
 jira_db_allocated_storage    = 100
 jira_db_iops                 = 1000
+jira_db_name                 = "jira"
```

Comment:
Is this fixing some e2e failure?

Reply:
Yes, it looks this is part of solution. I have the e2e tests green now for this branch.

Reply:
This is optional variable. In which part of the e2e test checks db_name?

Reply:
The default value is `null` and it means rds will be created with no database. I think we should change the default values of db_name for all products from null to the product name.